contracts/src/v0.8/tests/VRFConsumerV2Upgradeable.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/LinkTokenInterface.sol";
+import "../interfaces/VRFCoordinatorV2Interface.sol";
+import "../VRFConsumerBaseV2Upgradable.sol";
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+contract VRFConsumerV2Upgradeable is Initializable, VRFConsumerBaseV2Upgradeable {
+  uint256[] public s_randomWords;
+  uint256 public s_requestId;
+  VRFCoordinatorV2Interface public COORDINATOR;
+  LinkTokenInterface public LINKTOKEN;
+  uint64 public s_subId;
+  uint256 public s_gasAvailable;
+
+  function initialize(address _vrfCoordinator, address _link) public initializer {
+    __VRFConsumerBaseV2_init(_vrfCoordinator);
```

Comment:
Any reason for the underline prefix? Or just idiomatic.

Reply:
Yup exactly, just idiomatic.

Reply:
Basically to minimize the possibility of clashing with other people's contract initializers.